### PR TITLE
Allow disabling the scheduler via config param

### PR DESCRIPTION
**Pull request by cap10morgan**
This will allow us to scale scheduling but not sending ("scheduler"
is perhaps a bad name for that part of the system; it should probably be
renamed "sender").

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -22,9 +23,11 @@ wildfly:
     ADMIN_USERNAME: admin
     ADMIN_PASSWORD: admin
 datomic:
-  image: quay.io/democracyworks/datomic-tx:0.9.5327
+  image: quay.io/democracyworks/datomic-tx:0.9.5394
+  environment:
+    ALT_HOST: datomic
 rabbitmq:
-  image: rabbitmq:3.5.3-management
+  image: rabbitmq:3.6.2-management
```
Comment (danielglauser) [inline docker-compose.yml]: @cap10morgan Are we sure this is okay with the older version of Kehaar (0.4.0)?

Comment (cap10morgan) [inline docker-compose.yml]: Yes

**Comment by danielglauser** [comment]
LGTM

**Comment by danielglauser** [comment]
LEBTM (Looks Even Better To Me)

**Comment by danielglauser** [comment]
LGTM

**Pull request merged**